social_core/backends/open_id_connect.py

```diff
@@ -236,7 +236,9 @@ def validate_and_return_id_token(self, id_token, access_token):
 
         # pyjwt does not validate OIDC claims
         # see https://github.com/jpadilla/pyjwt/pull/296
-        if claims.get("at_hash") != self.calc_at_hash(access_token, key["alg"]):
+        if "at_hash" in claims and claims["at_hash"] != self.calc_at_hash(
+            access_token, key["alg"]
+        ):
             raise AuthTokenError(self, "Invalid access token")
```

Comment:
As far as I can tell, this isn't covered by the tests currently.

Reply:
Apparently, there is no test where at_hash claim would be missing, so this behavior is not tested (otherwise we would see a test failure earlier).